pkg/webui/components/unit-input/index.js

```diff
@@ -98,6 +98,7 @@ class UnitInput extends React.PureComponent {
       <React.Fragment>
         <div className={classnames(className, style.container)}>
           <Input
+            data-test-id="unit-input"
```

Comment:
If we have several of `UnitInput` component on the same view? `cy.findByTestId` will return all of them.